internal/scheduler/database/query/query.sql

```diff
@@ -53,7 +53,7 @@ UPDATE runs SET run_attempted = true WHERE run_id = ANY(sqlc.arg(run_ids)::UUID[
 UPDATE runs SET running = true WHERE run_id = ANY(sqlc.arg(run_ids)::UUID[]);
 
 -- name: MarkRunsCancelledByJobId :exec
-UPDATE runs SET cancelled = true WHERE job_id = ANY(sqlc.arg(job_ids)::text[]);
+UPDATE runs SET cancelled = true, terminated_timestamp = $1 WHERE job_id = ANY(sqlc.arg(job_ids)::text[]);
```

Comment:
I believe we can generalize this to all job runs marking functions ( MarkRunsFailed, MarkRunsSucceeded....). We don't a dedicated `SetX` function to update the timestamp for a jobrun, we can just embed the timestamp update into the status update like this one.

Reply:
Looks good to me.